ui/apps/profile/src/components/pages/profile-engagement/interests-page.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useMemo } from 'react';
+import { RootComponentProps } from '@akashaorg/typings/ui';
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+import Card from '@akashaorg/design-system-core/lib/components/Card';
+import Pill from '@akashaorg/design-system-core/lib/components/Pill';
+import Stack from '@akashaorg/design-system-core/lib/components/Stack';
+import Text from '@akashaorg/design-system-core/lib/components/Text';
+import { ProfileHeader } from '@akashaorg/design-system-components/lib/components/Profile';
+import {
+  useGetProfileByDidQuery,
+  useGetInterestsByDidQuery,
+} from '@akashaorg/ui-awf-hooks/lib/generated/hooks-new';
+import { getProfileImageVersionsWithMediaUrl, hasOwn, useGetLogin } from '@akashaorg/ui-awf-hooks';
+
+const InterestsPage: React.FC<RootComponentProps> = props => {
+  const { plugins } = props;
+  const { profileId } = useParams<{ profileId: string }>();
+  const { t } = useTranslation('app-profile');
+
+  const navigateTo = plugins['@akashaorg/app-routing']?.routing?.navigateTo;
+
+  const loginQuery = useGetLogin();
+
+  const profileDataReq = useGetProfileByDidQuery(
+    {
+      id: profileId,
+    },
+    {
+      select: response => response.node,
+      enabled: !!loginQuery.data?.id,
+    },
+  );
+
+  const interestsReq = useGetInterestsByDidQuery({ id: profileId });
+
+  const topics = useMemo(
+    () =>
+      interestsReq.data?.node && hasOwn(interestsReq.data?.node, 'interests')
+        ? interestsReq.data.node.interests.topics
+        : [],
+
+    [interestsReq.data],
+  );
+
+  if (!loginQuery.data?.id) {
+    return navigateTo({
+      appName: '@akashaorg/app-profile',
+      getNavigationUrl: () => `/${profileId}`,
+    });
+  }
+
+  const { isViewer, profile: profileData } = Object.assign(
+    { isViewer: null, profile: null },
+    profileDataReq.data,
+  );
+
+  const background = getProfileImageVersionsWithMediaUrl(profileData?.background);
+  const avatar = getProfileImageVersionsWithMediaUrl(profileData?.avatar);
+
+  return (
+    <Stack direction="column" spacing="gap-y-4" fullWidth>
+      <ProfileHeader
+        did={profileData?.did}
+        validAddress={true}
+        background={background}
+        avatar={avatar}
+        name={profileData?.name}
+        ensName={null}
+        viewerIsOwner={isViewer}
+        copyLabel={t('Copy to clipboard')}
+        copiedLabel={t('Copied')}
```

Comment:
We can make this props optional on `ProfileHeader` as we aren't actually using them on the interests page.